tests/net/lib/coap/src/main.c

```diff
@@ -718,16 +719,18 @@ static int test_block1_request(struct coap_block_context *req_ctx, u8_t iter)
 
 	/* In last iteration "current" field not updated */
 	if (iter < 4) {
-		if (req_ctx->current !=
-		    coap_block_size_to_bytes(COAP_BLOCK_32) * iter) {
+		current_block = coap_block_size_to_bytes(COAP_BLOCK_32) * iter;
+
```

Comment:
you could avoid the empty line before the if since if conditional statement is testing current_block (as everywhere below as well)

Reply:
ok

Reply:
Done. updated the PR.